tests/tools/test_slang.py

```
@@ -540,3 +554,888 @@ def test_pyslang_syntax_printer():
+    top = Design("leaftop")
+    top.set_dataroot("root", os.getcwd())
+    with top.active_fileset("rtl"), top.active_dataroot("root"):
+        top.set_topmodule("leaftop")
+        top.add_file("leaftop.v")
+        top.add_depfileset(leaf, depfileset="rtl", fileset="rtl")
+    return top, leaf
+
+
+def test_uniquified_unparameterized_module():
+    """An unparameterized module is hardened directly: no wrapper, no variant src."""
+    top, leaf = _leaf_designs()
```

Ruff (0.15.21)

[warning] Unpacked variable `leaf` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)